client/app/main/student/student.controller.js

```diff
@@ -13,6 +14,11 @@ function StudentCtrl($scope, $state, $stateParams, Student, toastr) {
     });
   });
 
+  AbsenceRecord.student({id: $stateParams.id})
+    .$promise.then(function(total) {
+      $scope.percentage = Math.floor((total.present / total.enrolled) * 100);
+    });
+
```

Comment:
I feel like this should go somewhere else. Just not quite sure where. Possibly refactor into student service?